### PR TITLE
Build a query parameter to make unav version configurable

### PR DESCRIPTION
**Pull request by bandana147**
* Adds a query parameter to unav where one can pass the version of unav to debug

Resolves: [MWPW-147126](https://jira.corp.adobe.com/browse/MWPW-147126)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.live/drafts/blaishram/unav-test
- After: https://gnav--milo--adobecom.hlx.live/drafts/blaishram/unav-test?unav=1.0&martech=off

### CODE REVIEW

**Review by sonawanesnehal3 — APPROVED** [review]

**Review by nishantka — APPROVED** [review]

**Review by Deva309 — APPROVED** [review]

**Review by sharmrj — APPROVED** [review]

**Review by mokimo — APPROVED** [review]

File: libs/blocks/global-navigation/global-navigation.js [hunk 1]
```diff
@@ -512,9 +512,10 @@ class Gnav {
       return 'linux';
     };
 
+    const unavVersion = new URLSearchParams(window.location.search).get('unav');
```
Comment (narcis-radu) [inline libs/blocks/global-navigation/global-navigation.js:515]: This should be easier
```suggestion
    const unavVersion = new URLSearchParams(window.location.search).get('unav') || '1.1';
```

**Review by overmyheadandbody — COMMENTED** [review]
I agree with @narcis-radu's suggestion. Additionally, I'd argue `unav` is not very descriptive. I'd expect to be able to turn a feature on or off with such wording. I'd adapt the name to reflect that the value is impacting the version.

**Comment by sigadamvenkata** [comment]
verified the change in Gnav branch. now Unav JS , Css and other libraries are rendering as per the unav version parameter passed in url as query parameter. PFB
image
image

@bandana147 @salonijain3

**Comment by bandana147** [comment]
> I agree with @narcis-radu's suggestion. Additionally, I'd argue `unav` is not very descriptive. I'd expect to be able to turn a feature on or off with such wording. I'd adapt the name to reflect that the value is impacting the version.

@overmyheadandbody Does `unavVersion` sounds good?

**Review by overmyheadandbody — APPROVED** [review]
Hey @bandana147! Yes, this reflects the purpose much better, thanks for integrating it!

**Comment by sigadamvenkata** [comment]
image
image

**Pull request merged**